pkg/gen/input/makefile/internal/file/makefile_gen_go_mk.go

```diff
@@ -35,8 +35,9 @@ SOURCES        := $(shell find . -name '*.go')
 VERSION        := $(shell architect project version)
 LDFLAGS        ?= -w -linkmode 'auto' -extldflags '-static' \
   -X '$(shell go list .)/pkg/project.buildTimestamp=${BUILDTIMESTAMP}' \
-  -X '$(shell go list .)/pkg/project.gitSHA=${GITSHA1}'
-.DEFAULT_GOAL := build
+  -X '$(shell go list .)/pkg/project.gitSHA=${GITSHA1}' \
+  -X '$(shell go list .)/pkg/project.version=${VERSION}'
+  .DEFAULT_GOAL := build
```

Comment:
This is a Makefile spaces are important here :) 

```suggestion
.DEFAULT_GOAL := build
```